src/modules/Dashboard/api/api.ts

```diff
@@ -994,5 +994,5 @@ export const disableIntegrationApi = (integrations: string[], signal?: AbortSign
     signal,
   };
 
-  return authApiClient.delete('/integrations/', config);
+  return authApiClient.delete('/integrations', config);
```

Comment:
same for this endpoint, though the integration is not part of this PR so it's ok if you keep it for now